main.go

```diff
@@ -1,7 +1,46 @@
 package main
 
-import "fmt"
+import (
+	"net/http"
+	"os"
+
+	"github.com/Gamebuildr/Dave/client"
+	"github.com/Gamebuildr/Dave/pkg/config"
+	"github.com/Gamebuildr/Dave/pkg/scaler"
+)
 
 func main() {
-	fmt.Printf("Hello from Dave")
+	cli := client.DaveClient{}
```

Comment:
picky but maybe rename as cli give inferences that it's something to do with running commands on the command line